kitsune/sumo/templatetags/jinja_helpers.py

```diff
@@ -211,6 +211,14 @@ def datetimeformat(context, value, format='shortdatetime'):
     Returns a formatted date/time using Babel's locale settings. Uses the
     timezone from settings.py, if the user has not been authenticated.
     """
+
+    def get_user_timezone(user):
```

Comment:
Why did you move this code into an inner function? It seems a bit oddly placed, and doesn't really help with the readability.